packages/mdc-checkbox/foundation.ts

```diff
@@ -48,12 +48,14 @@ export class MDCCheckboxFoundation extends MDCFoundation<MDCCheckboxAdapter> {
       isIndeterminate: () => false,
       removeClass: () => undefined,
       removeNativeControlAttr: () => undefined,
+      setChecked: () => undefined,
       setNativeControlAttr: () => undefined,
       setNativeControlDisabled: () => undefined,
     };
   }
 
   private currentCheckState_ = strings.TRANSITION_STATE_INIT;
+  private previousIsChecked_ = false;
```

Comment:
I think we should rename this to `realCheckedState_` or `inputCheckedState_`, since that will be a more accurate description of what this variable indicates